src/main/java/com/tada/summerboot/controller/MyErrorController.java

```diff
@@ -0,0 +1,36 @@
+package com.tada.summerboot.controller;
+
+import org.springframework.boot.web.servlet.error.ErrorController;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.RequestDispatcher;
+import javax.servlet.http.HttpServletRequest;
+
+@Controller
+public class MyErrorController implements ErrorController {
+
+    @RequestMapping("/error")
+    public String handleError(HttpServletRequest request) {
+        Object status = request.getAttribute(RequestDispatcher.ERROR_STATUS_CODE);
+        //do something like logging
+
+        if (status != null) {
+            int statusCode = Integer.parseInt(status.toString());
```

Comment:
1. can we have the following code that returns the exact error code? #TwoPunchSolution

```
        if (status != null) {
            return "errors/" + status.toString();
         }
```

Reply:
2. Please create under 

`resources/templates/errors/404.html`

`resources/templates/errors/500.html`